pkg/handler/metadata/metadata_test.go

```diff
@@ -0,0 +1,57 @@
+package metadata_handler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dwarvesf/fortress-api/pkg/config"
+	"github.com/dwarvesf/fortress-api/pkg/logger"
+	"github.com/dwarvesf/fortress-api/pkg/service"
+	"github.com/dwarvesf/fortress-api/pkg/store"
+)
+
+func TestHandler_GetChains(t *testing.T) {
```

Comment:
It should be GetWorkingStatus and update the same for keyword "chain" in this file